config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I don't know what this does. Maybe just delete it?

Reply:
Huh, yeah. If you delete it and everything seems fine then blow it away.